NDB/clusterj-fixes/clusterj-fix.patch

```diff
@@ -86,6 +94,12 @@ index 833acfac..a26e0a3d 100644
 -        String className = element.getClassName();
 -        String methodName = element.getMethodName();
 -        delegate.logp(Level.FINER, className, methodName, message);
++        // Throwable t = new Throwable();
```

Comment:
Same here

Reply:
As discussed earlier the code in file "NDB/clusterj-fixes/clusterj-fix.patch" is reorganized but it is exactly the same as before.